app/controllers/dradis/plugins/engines_controller.rb

```diff
@@ -0,0 +1,33 @@
+module Dradis::Plugins
```

Comment:
I don't think enabling/disabling is a dradis-plugins thing rather it should be part of dradis-pro. dradis-plugins has never had a standalone controller and adding one for it now doesn't have much merit.